.github/workflows/sync_issues_to_azdo.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync issue to Azure DevOps work item
+
+on:
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned]
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danhellem/github-actions-issue-to-work-item@master
+        env:
+          ado_token: "${{ secrets.AZDO_WORK_ITEM_TOKEN }}"
+          github_token: "${{ secrets.GH_REPO_TOKEN }}"
```

Comment:
These secrets are tokens managed at the ni organizational level on GitHub. They are only available to repos in the ni organization and are maintained by the internal team that manages our GitHub accounts/repos.